src/common/components/ui/BorderedIcon/BorderedIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { DownloadIcon } from '@patternfly/react-icons';
+import { global_Color_200 as greyColor } from '@patternfly/react-tokens';
+
+const style = {
+  padding: '.3em',
+  borderRadius: '50%',
+  border: '1px solid',
+  borderColor: greyColor.value,
+  boxSizing: 'content-box',
+  verticalAlign: '-1.125em',
```

Comment:
I checked how patternfly determines vertical align per icon size and for our icon size (including border) the vertical align should be `-0.3em;`

Reply:
To make the component universal for all icon sizes we should calculate the padding based on the `size` prop which we add to this component (sm, md, lg, xl) Similar to [1]. Same for the verticalAlign.

[1] https://github.com/patternfly/patternfly-react/blob/main/packages/react-icons/src/createIcon.tsx#L10